src/foam/nanos/ticket/Relationships.js

```diff
@@ -10,7 +10,7 @@ foam.RELATIONSHIP({
   unauthorizedTargetDAOKey: 'localTicketDAO',
   // REVIEW: unable to get owner to display in any section on Ticket.
   targetProperty: {
-    readVisibility: 'RO',
+    visibility: 'RO',
```

Comment:
Would this not prevent us from updating or setting ticket owners? I have a suspicion the RichChoiceView is not aware of visibility related modes if this being set enables us to make changes to the owner on edit or create.

Reply:
hmm, I expected the visibility function to apply to create read and update operations but I guess not. Seems kinda odd.

"In regards to update/edit ticket, you are right this will prevent us from editing the owner on a ticket. Is this functionality that we need?"

Looking at the property label it leads me to think that I'd be granting a user to sudo as the user selected under "sudo as User" but I don't see where we select the user sudoing into the user. If it's to select the user to sudo as admin or another user with greater permissions then we probably wouldn't.

Reply:
@jlhughes